gokay/bcp47.go

```diff
@@ -16,3 +16,18 @@ func IsBCP47(s *string) error {
 
 	return err
 }
+
+func IsBCP47OrEmpty(s *string) error {
+	if s == nil || *s == "" {
```

Comment:
Thoughts about moving this validation to the original `IsBCP47` function and implementing a `NotEmpty` validation in the future if needed?